src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -44,3 +44,7 @@ target_include_directories(t_uptane_key PUBLIC ${PROJECT_SOURCE_DIR}/src/virtual
 target_link_libraries(t_uptane_key virtual_secondary)
 
 aktualizr_source_file_checks(${SOURCES} ${HEADERS} ${TEST_SOURCES})
+
+set_target_properties (t_aktualizr PROPERTIES COMPILE_FLAGS -pthread LINK_FLAG -pthread)
+set_target_properties (t_aktualizr_fullostree PROPERTIES COMPILE_FLAGS -pthread LINK_FLAGS -pthread)
+set_target_properties (t_uptane_key PROPERTIES COMPILE_FLAGS -pthread LINK_FLAGS -pthread)
```

Comment:
I would prefer if we didn't have to have this hardcoded and duplicated for all these targets. Why is this required?

Reply:
We have a few targets which use libfiu, for these targets, we get linking error " undefined reference to pthread_atfork" I think it related to GCC version there are some differences in -lpthread and -pthread

Reply:
I had installed GCC 7 on ubuntu xenial and the result is the same as for gcc 5, which is the default for the xenial, so the issue is not related to GCC version...